render/include/mainframe/render/color.h

```diff
@@ -69,6 +69,30 @@ namespace mainframe {
 				}
 			}
 
+			static ColorType debug(int index) {
+				ColorType cols[]{
+					ColorType(255, 82, 82, 255),// rgb(255, 82, 82)
```

Comment:
I left the rgb comment so we can see what color it is:
![](https://i.rawr.dev/2102Q0Xkvc.png)